app.js

```diff
@@ -137,38 +138,167 @@ class Products {
         });
       });
     }
-    setCartValues(cart) {
-      let tempTotal = 0;
-      let itemsTotal = 0;
-      //for the whatsapp checkout link
-      let linkValue = `https://wa.me/+2349032592825?text=I%20will%20like%20to%20place%20an%20order%20of%20`;
-      //when you click on the add to cart it will display the total price in the cart after adding them up.
-      cart.map((item) => {
-        tempTotal += item.price * item.amount;
-       itemsTotal += item.amount;
-       //for displaying the amount of items and total price.
-       linkValue += item.amount + ' ' + item.title + ', ';
-      });
-      cartTotal.innerText = parseFloat(tempTotal.toFixed(2));
-      cartItems.innerText = itemsTotal;
-      //to create href attributes to display the link contents and items total price rounded up to 2.
-      document.getElementById("check").href= linkValue + 'Total Price:$' + parseFloat(tempTotal.toFixed(2));
-    }
-    addCartItems(item) {
-      const div = document.createElement("div");
-      div.classList.add("cart-item");
-      div.innerHTML = `<img src=${item.image} alt="product" />
-          <div class="text-black">
-            <h4 class="capitalize">${item.title}</h4>
-            <h5>$${item.price}</h5>
-            <span class="remove-item cursor-pointer" data-id = ${item.id}>remove</span>
-          </div>
-          <div class="text-black"> 
-          <i class="fas fa-chevron-up cursor-pointer" data-id= ${item.id}></i>
-          <p class="item-amount self-center">${item.amount}</p>
-          <i class="fas fa-chevron-down cursor-pointer" data-id= ${item.id}></i>
+
+    //set cart values plus the Checkout functions.
```

Comment:
since they are all in the cart, why can't I have them in a function? The set cart values hold the parameters to place the order too.. Soo from the value it is getting it is using that to send the number of item in the cart.

Reply:
Ok will create a function for them

Reply:
Done this.